### PR TITLE
2018 releases

### PR DESCRIPTION
**Pull request by josephholsten**

### CODE REVIEW

**Comment by damacus** [comment]
- major releases
- of note
- Deprecated
- Now in Chef

File: content/posts/2018-12-31-releases.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+## Deprecated cookbooks
+
+We've deprecated the mac_os_x cookbook in favour of the excellently written [MacOS cookbook from Microsoft](https://github.com/Microsoft/macos-cookbook).
+
+[Chef Compliance](https://docs.chef.io/chef_compliance.html) was deprecreated as it's encorporated into Chef Automate. Our accompanying [cookbook](http://supermarket.chef.io/cookbooks/chef-compliance) has been deprecated.
+
+## Too Many To Note
+
+We've done a lot of releaes in the past year, here's an absolutely huge list of all the "minor" releases.
```
Comment (shoekstra) [inline content/posts/2018-12-31-releases.md]: typo in "releaes"

File: content/posts/2018-12-31-releases.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+- [`Redisio`](https://supermarket.chef.io/cookbooks/redisio) - Major release & new maintainers! ([@tas50](https://github.com/tas50)/[@majormoses]https://github.com/majormoses)/[@shoekstra](https://github.com/shoekstra))
+- [`vscode`](https://supermarket.chef.io/cookbooks/sc_vscode) - First version of the [vscode cookbook](https://github.com/sous-chefs/vscode) was ([@Xorima](https://github.com/Xorima))
+- [`sc-chruby`](https://supermarket.chef.io/cookbooks/sc-chruby) Released late in 2018 by [@damacus](https://github.com/damacus)
+
+## Adoptions
+
+- [`java`](https://supermarket.chef.io/cookbooks/java)
+- [`stunnel`](https://supermarket.chef.io/cookbooks/stunnel)
+- [`squid`](https://supermarket.chef.io/cookbooks/squid)
+- [`php`](https://supermarket.chef.io/cookbooks/php)
```
Comment (shoekstra) [inline content/posts/2018-12-31-releases.md:30]: Can we sort this list?

File: content/posts/2018-12-31-releases.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+The following cookbooks are currently underloved, but with a small number of community members. We know they've provided value to poeple in the past, but require some love.
+
+- [`nexus`](www.github.com/sous-chefs/nexus)
+- [`aptly`](www.github.com/sous-chefs/aptly)
+- [`winrm`](www.github.com/sous-chefs/winrm)
+- [`kismet`](www.github.com/sous-chefs/kismet)
+- [`sensors`](www.github.com/sous-chefs/sensors)
+- [`reprepro`](www.github.com/sous-chefs/reprepro)
+- [`hashicorp-vault`](www.github.com/sous-chefs/hashicorp-vault)
+- [`consul`](www.github.com/sous-chefs/consul)
```
Comment (shoekstra) [inline content/posts/2018-12-31-releases.md]: And this list?

File: content/posts/2018-12-31-releases.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+- [`scmongodb`](https://supermarket.chef.io/cookbooks/sc-mongodb)
+- [`pulledpork`](https://supermarket.chef.io/cookbooks/pulledpork)
+- [`unifi`](https://supermarket.chef.io/cookbooks/unifi)
+- [`mysql`](https://supermarket.chef.io/cookbooks/mysql)
+- [`mariadb`](https://supermarket.chef.io/cookbooks/mariadb)
+- [`vagrant`](https://supermarket.chef.io/cookbooks/vagrant)
+- [`nagios`](https://supermarket.chef.io/cookbooks/nagios)
+- [`filesystem`](https://supermarket.chef.io/cookbooks/filesystem)
+- [`openvpn`](https://supermarket.chef.io/cookbooks/openvpn)
+- [`dhcp`](https://supermarket.chef.io/cookbooks/dhcp)
```
Comment (shoekstra) [inline content/posts/2018-12-31-releases.md]: And this list?

File: content/posts/2018-12-31-releases.md (hunk 2 above)
Comment (shoekstra) [reply content/posts/2018-12-31-releases.md:30]: This list feels quite small, I'm sure we picked up more?  

You can add the confluence cookbook (https://github.com/sous-chefs/confluence) so long.

**Review by shoekstra — DISMISSED** [review]

File: content/posts/2018-12-31-releases.md (hunk 3 above)
Comment (damacus) [reply content/posts/2018-12-31-releases.md]: Going to order this by most want fixed first 👍

File: content/posts/2018-12-31-releases.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+type = "post"
++++
+
+Hello, this is Joseph Holsten & Dan Webb from Sous Chefs. We've had quite a year, so I thought I'd summarize the major moments. Here we go!
+
+# 2018 In Review
+
+2018 was a pretty big year in Sous-Chefs, we started on [OpenCollective](https://opencollective.com/sous-chefs) to raise funds for our core contributors. Many hours go into rewriting, maintaining, and reviewing pull requests. It's a great way to give back to those that empower you and your company!
+
+We moved out cookbook testing from TravisCI to CircleCI accross the board, for reliability and speed reasons. We were one of the first organisatons to adopt [Orbs](https://circleci.com/orbs/) a way of reusing commands and more importantly yaml without losing the expressiveness that comes with CircleCI. Check ours out [here](https://github.com/sous-chefs/orbs) and [here].
```
Comment (xorima) [inline content/posts/2018-12-31-releases.md]: and [here] no link

File: content/posts/2018-12-31-releases.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+With the blatent plugs out of the way, onto the cookbooks.
+
+## Major/New Releases
+
+- [`Postgresql`](https://github.com/sous-chefs/postgresql/blob/master/CHANGELOG.md) - The biggest release of the year was . Completelty rewritten with custom resources and a full unit and functional test suite. ([@damacus](https://github.com/damacus)/[@martinisoft](https://github.com/martinisoft))
```
Comment (JohnRoesler) [inline content/posts/2018-12-31-releases.md]: `was . Completely` something missing there?

File: content/posts/2018-12-31-releases.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+- [`Redisio`](https://supermarket.chef.io/cookbooks/redisio) - Major release & new maintainers! ([@tas50](https://github.com/tas50)/[@majormoses]https://github.com/majormoses)/[@shoekstra](https://github.com/shoekstra))
+- [`vscode`](https://supermarket.chef.io/cookbooks/sc_vscode) - First version of the [vscode cookbook](https://github.com/sous-chefs/vscode) was ([@Xorima](https://github.com/Xorima))
```
Comment (JohnRoesler) [inline content/posts/2018-12-31-releases.md]: `was 'released by?' @Xorima`

**Comment by damacus** [comment]
@josephholsten shall we release this soon? 🤔

**Review by damacus — APPROVED** [review]

**Comment by josephholsten** [comment]
🎉

**Pull request merged**